### ISSUE TITLE
Add Hubot to the group

### ISSUE THREAD

**Issue by khanly** [athieriot/hubot-yammer#27]
export HUBOT_YAMMER_GROUPS="groups list"

How do I add new group(s) for hubot to recognize and be in the group?  I do not want to stop hubot, add new group to HUBOT_YAMMER_GROUPS, and restart hubot. 

Thanks!

**Comment by athieriot** [comment]
Hi @khanly,

I am afraid you will have to restart Hubot before any new group is taken into account.
Is this really a big issue?

Aurélien

**Comment by khanly** [comment]
Yes, I am trying to figure out a way for hubot to be active in the group instantly. Restart hubot will probably affect other groups and people have to wait for the next restart to have hubot in the group.

**Comment by athieriot** [comment]
All right.
After checking, I am afraid it's not possible at the moment as the Group Ids are being resolved when Hubot starts.

However, it shouldn't be too difficult to add support for dynamic group join if you fancy giving it a go?

@emedvedev and I would be happy to review your work.

P.S: Would you mind telling us a bit more about your use case as well? I am curious to know how you are using Hubot !
